lib/json.lua

```diff
@@ -167,6 +168,7 @@ end
 function decode_scanArray(s,startPos)
   local array = {}	-- The return value
   local stringLen = string.len(s)
+  local object
```

Comment:
It is necessary because the function json.decode return two values, the first one is a json map with lua table and the second one an iterator.
This method is recursive, if use 'local' before the method call, the iterator(starPos) will be create a variable with the same name.
The local object is required to doesn't create global variables in the package execution.

Reply:
Ok.